### DISCUSSION TITLE
Human Feedback

### DISCUSSION THREAD

**Discussion by simonhalle** [truera/trulens#550] [Q&A] [answered]
Is there a way (or do you plan on adding it) to enter human feedback from the Trulens UI? For example adding to LLM-based based feedback for a given question-answer (or commenting those feedback) from the feedback panel and adding it to the dataset. Thanks.

**Comment by joshreini1** [comment, marked as answer]
Hey @simonhalle - you can add human feedback with TruLens using `session.add_feedback`.

Check out our quickstart with this workflow: 

https://www.trulens.org/trulens/getting_started/quickstarts/human_feedback/
